fix: spacing between bottom of related content and footer. [NP-1029]

baseonmars opened the pull request:
Fix the spacing between related content, the toolbar and footer. This should be 40px (2.5rem) at the smallest breakpoint.

Spacing between the two titles and each list item is already 16px/1rem - assuming this was fixed in a previous PR related to typography gap analysis?

**With toolbar, smallest/mobile breakpoint (logged in):**

(will need additional PR on public-repo to remove top padding from toolbar)

<img width="744" alt="Screenshot 2020-11-04 at 15 00 40" src="https://user-images.githubusercontent.com/45935/98127356-82a61f80-1eae-11eb-8205-8afa96c12d86.png">

**Logged out, smallest/mobile breakpoint (logged out):**

<img width="742" alt="Screenshot 2020-11-04 at 14 59 31" src="https://user-images.githubusercontent.com/45935/98127223-5a1e2580-1eae-11eb-80df-d4c2caae79f0.png">

**Advisernet related content**

This change also affects advisernet related content - spacing on the list is now a uniform 1rem/16px (VRTs updated to reflect this)

<img width="155" alt="Screenshot 2020-11-04 at 16 48 42" src="https://user-images.githubusercontent.com/45935/98140798-c5232880-1ebd-11eb-97d9-bad94cab27ee.png">

baseonmars commented:
related: https://github.com/citizensadvice/public-website/pull/488

davidsauntson reviewed (commented):
I think the VRTs will pick this change up

baseonmars commented:
I've updated the VRTs - they did highlight a problem with my initial PR. I fixed that and realised there was an opportunity to normalise the spacing around the Adviser content in the advisernet related content - so made that change and updated VRTs.

davidsauntson reviewed (approved):
🌞 nice

davidsauntson commented on scss/6-components/advisernet/_related-content.scss:
```diff
@@ -9,6 +9,10 @@
     }
 
+    ul {
+      margin-bottom: 1rem;
```
lol, should be a var

baseonmars replied:
fml... yes.

davidsauntson replied:
<3
